### ISSUE TITLE
Problems related to the calculation of the lateral pressure profile

### ISSUE THREAD

**Issue by ohsOllila** [davesmith4398/best_practice_membranes#4]
Lateral pressure calculations are mentioned as a method to validate simulations as well as method to calculate elastic properties. The correct calculation method of lateral pressure profiles is, however,
not agreed in the literature. First of all, some authors are not convinced that the local pressure would be well defined quantity due to the ambiguity related to the choise of contour along which the forces act across the space. Another issue has been the composition used to divide forces of many-body interactions to the pair-wise interactions. Originally, the so called Goetz-lipowsky decomposition (GLD) was used. Later on, it was argues that the so called central force decomposition (CFD) gives
conservation of angular momentum. These compositions result to different pressure profiles, and presumably different elastic constants, in atomistic simulations where the pressure from dihedrals with four body interactions gives a significant contribution. Comparison between the pressures arising
from Ryckaert-Belleman dihedrals from Berger lipid bilayer simulation with different decompositions is shown below.

![methoddihcomp](https://user-images.githubusercontent.com/7114769/42395087-3043e9b8-8165-11e8-91b6-03bc27d03b9f.png)

For me the GLD result is intuitive reasonable because dihedrals give small negative pressure along the membrane. This can be understood as the tendency of acyl chains to go to all trans state, which is
the energy of minimum of the dihedrals and has lower area than bilayer in equilibrium, therefore generating negative pressure. For me it is hard to understand the physical origin of large peaks
from CFD decomposition. The authors, however, argue that it is more realistic due to the conservation of angular momentum. 

In conclusion, I think that the calculation methods of the lateral pressure profile are not sufficiently robust to be used to validate simulation results, at least not from atomistic models.
